microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/MetricsTest.java

```diff
@@ -55,15 +55,12 @@ public static WebArchive createDeployment() {
 
     @Before
     public void prepare() {
+        String metricsURL = "http://" + managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort() + "/metrics";
```

Comment:
Looking at it I would prefer taking those values from `ArquillianContainerProperties` like:
```
ArquillianContainerProperties arquillianContainerProperties = new ArquillianContainerProperties(ArquillianDescriptorWrapper.getArquillianDescriptor());
String metricsURL = "http://" + arquillianContainerProperties.getDefaultManagementAddress() + ":" + arquillianContainerProperties.getDefaultManagementPort() + "/metrics";
```

afaict injecting of `managementClient` in:
```
@ContainerResource
ManagementClient managementClient;
```
causes that there is created management connection to server just to get management address and port which is somewhat misusing this object. wdyt?

Reply:
This PR is to reduce current usage of `managementClient`

ArquillianContainerProperties can by next step, but I wouldn't connect it to this PR.
ArquillianContainerProperties depends on what is defined in xml files + hard-coded defaults.
WF TS uses `managementClient` approach and no problems were noticed.

We should agree on preferred approach because currently we have mixture of both approaches. Please create issue for this + drive the discussion.
```
git grep getDefaultManagementAddress
microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/tools/HealthUrlProvider.java:        return "http://" + arqProps.getDefaultManagementAddress() + ":" + arqProps.getDefaultManagementPort() + "/health";
microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/tools/HealthUrlProvider.java:        return "http://" + arqProps.getDefaultManagementAddress() + ":" + arqProps.getDefaultManagementPort()
microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/tools/HealthUrlProvider.java:        return "http://" + arqProps.getDefaultManagementAddress() + ":" + arqProps.getDefaultManagementPort()
microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/integration/restclient/RestClientIntegrationTest.java:                "services.provider.host=" + arquillianContainerProperties.getDefaultManagementAddress()
tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/arquillian/ArquillianContainerProperties.java:    public String getDefaultManagementAddress() throws ConfigurationException {
tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/creaper/ManagementClientProvider.java:                        arquillianContainerProperties.getDefaultManagementAddress(),
```
```
git grep getMgmtAddress
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/ConnectionStressMetricsTest.java:        metricsURL = new URL("http://" + managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort() + "/metrics");
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/ConnectionStressMetricsTest.java:                    .baseUri("http://" + managementClient.getMgmtAddress())
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/InvalidMimeTypeMetricsTest.java:                .baseUri("http://" + managementClient.getMgmtAddress())
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/MetricsTest.java:                .baseUri("http://" + managementClient.getMgmtAddress())
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/MetricsTest.java:                .baseUri("http://" + managementClient.getMgmtAddress())
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/NamefellowMetricsTest.java:                .baseUri("http://" + managementClient.getMgmtAddress())
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/NamefellowMetricsTest.java:                .baseUri("http://" + managementClient.getMgmtAddress())
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/integration/config/CustomMetricBaseTest.java:        metricsURL = "http://" + managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort() + "/metrics";
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/integration/config/CustomMetricBaseTest.java:                .baseUri("http://" + managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort() + "/metrics")
microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/integration/ft/FTMetricTest.java:                .baseUri("http://" + managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort() + "/metrics")
```

Reply:
here we go https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues/139